assets/stylesheets/sass/main/main.scss

```diff
@@ -759,9 +765,13 @@ figure {
 }
 
 .why-voteusa {
-  padding: 3em 0;
+  padding: 3em 0 4em;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets